notebooks/ICESat-2_MODIS_Arctic_Sea_Ice/Customize and Access Data.ipynb

```diff
@@ -218,11 +222,10 @@
     "    # Create session to store cookie and pass credentials to capabilities url\n",
     "    session = requests.session() \n",
     "    s = session.get(capability_url)\n",
-    "    response = session.get(s.url,auth=(uid,pswd))\n",
+    "    response = session.get(s.url, headers={\"Authorization\": f\"Bearer {auth}\"})\n",
```

Comment:
```suggestion
    "",
```

Reply:
I tested that we don't actually need this response line - we can just pass `s` into the function below.